tst/providers/MeetingProvider/MeetingManager.test.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import '@testing-library/jest-dom';
+
+import {
+  ConsoleLogger,
+  DefaultActiveSpeakerPolicy,
+  DefaultDeviceController,
+  DefaultMeetingSession,
+  LogLevel,
+  MeetingSessionConfiguration,
+  MeetingSessionPOSTLogger,
+  MultiLogger,
+} from 'amazon-chime-sdk-js';
+
+import { MeetingManager } from '../../../src/providers/MeetingProvider/MeetingManager';
+import { MeetingManagerJoinOptions } from '../../../src/providers/MeetingProvider/types';
+
+describe('Meeting Manager', () => {
+  let mockMeetingManagerJoinOptions: MeetingManagerJoinOptions;
+  let mockMeetingSessionConfiguration: MeetingSessionConfiguration;
+  let meetingManager: MeetingManager;
+  // eslint-disable-next-line
+  const GlobalAny = global as any;
+
+  beforeEach(() => {
+    // @ts-ignore
+    MeetingSessionConfiguration = jest.fn().mockImplementation(() => {});
+    mockMeetingSessionConfiguration = new MeetingSessionConfiguration();
+    mockMeetingManagerJoinOptions = {
+      logLevel: LogLevel.WARN,
+    };
+    meetingManager = new MeetingManager();
+    // @ts-ignore
+    MultiLogger = jest.fn().mockReturnValue({});
+    // @ts-ignore
+    MeetingSessionPOSTLogger = jest.fn().mockReturnValue({});
+    // @ts-ignore
+    ConsoleLogger = jest.fn().mockReturnValue({});
+    // @ts-ignore
+    DefaultDeviceController = jest.fn().mockReturnValue({});
+    // @ts-ignore
+    DefaultMeetingSession = jest.fn().mockReturnValue({
+      audioVideo: {
+        addObserver: jest.fn().mockReturnValue({}),
+        removeObserver: jest.fn().mockReturnValue({}),
+        listAudioInputDevices: jest.fn().mockReturnValue({}),
+        listVideoInputDevices: jest.fn().mockReturnValue({}),
+        listAudioOutputDevices: jest.fn().mockReturnValue({}),
+        chooseAudioInputDevice: jest.fn().mockReturnValue({}),
+        setDeviceLabelTrigger: jest.fn().mockReturnValue({}),
+        subscribeToActiveSpeakerDetector: jest.fn().mockReturnValue({}),
+      },
+      eventController:  {
+        addObserver: jest.fn().mockReturnValue({}),
+        removeObserver: jest.fn().mockReturnValue({}),
+      },
+    });
+    GlobalAny.navigator = jest.fn().mockReturnValue({
+      mediaDevices: jest.fn().mockReturnValue({
+        enumerateDevices: jest.fn().mockReturnValue(() => {}),
+        getUserMedia: jest.fn().mockReturnValue({
+          some: () => {}
+        }),
+      })
+    });
+  });
+
+  describe('join', () => {
+    it('should call parseJoinParams', async () => {
+      // @ts-ignore
+      const spy = jest.spyOn(meetingManager, 'parseJoinParams');
+      await meetingManager.join(mockMeetingSessionConfiguration, mockMeetingManagerJoinOptions);
+      // @ts-ignore
+      expect(spy).toHaveBeenCalledTimes(1);
+    });
+
+    it('should call createLogger if logger is not passed via params', async () => {
```

Comment:
This should just be checking on meeting session object logger property and verify that the default logger is created.